core/translations/cs.json

```diff
@@ -33,7 +33,7 @@
   },
   "header": {
     "language": "cs-CZ",
-    "version": "2.8.0"
+    "version": "2.8.1"
```

Comment:
Shouldn't it be 2.8.0 still?

Reply:
nah, we need the translations matched to the current in-tree version.

we can generate 2.8.0 blobs regardless of what is written here